components/airtable/airtable.app.mjs

```diff
@@ -22,7 +22,7 @@ export default {
     tableId: {
       type: "string",
       label: "Table",
-      description: "The table ID. If referencing a **Base** dynamically using data from another step (e.g., `{{steps.mydata.$return_value}}`), automatic table options won't work when configuring this step. Please enter a custom expression to specify the **Table**.",
+      description: "The table ID. If referencing a **Base** dynamically using data from another step (e.g., {{steps.trigger.event.metadata.baseId}}), you will not be able to select from the list of Tables, and automatic table options will not work when configuring this step. Please enter a custom expression to specify the **Table**.",
```

Comment:
nit, but is there a reason you removed the code formatting?

Reply:
Ah, sorry I missed that - my paste missed the `. Fixing.